__tests__/app.test.js

```diff
@@ -43,26 +43,36 @@ describe("/api/topics", () => {
   });
 });
 
-describe("/api/topics", () => {
-    test("GET 200 - should receive an array of topic objects contains slug and description keys with strings as values", () => {
+describe("/api/articles/:article_id", () => {
+    test("GET 200 - should receive an article object with the correct article_id", () => {
       return request(app)
-        .get("/api/topics")
+        .get("/api/articles/4")
         .expect(200)
         .then((result) => {
-          expect(Array.isArray(result.body.topics)).toBe(true);
-          expect(result.body.topics.length).toBe(3);
-          result.body.topics.forEach((topic)=>{
-            expect(typeof topic.slug).toBe('string');
-            expect(typeof topic.description).toBe('string');
-          })
+          expect(typeof result.body.article.article_id).toBe('number');
```

Comment:
Do you want the article to be any number? You should be more specific here to make sure you are returning the correct article